packages/apputils/src/toolbar.tsx

```diff
@@ -417,17 +451,28 @@ namespace ToolbarButton {
     /**
      * The callback for a click event.
      */
-    onClick?: () => void;
+    onChange: (event?: React.ChangeEvent<HTMLSelectElement>) => void;
+
+    /**
+     * The callback for a click event.
+     */
+    onKeyDown?: (event?: React.KeyboardEvent<HTMLSelectElement>) => void;
 
     /**
      * The class name added to the button.
      */
-    className?: string;
+    className: string;
 
     /**
-     * The tooltip added to the button node.
+     * The list of options to add to select.
      */
-    tooltip?: string;
+    options: string[];
+
+    /**
+     * The selected option value.
+     */
+    selected?: string;
+    [key: string]: any;
```

Comment:
Do we need this? seems like we could use some type checking safety by allowing any value as an option